cluster/hooks.go

```diff
@@ -51,6 +51,11 @@ import (
 	pkgHelmRelease "k8s.io/helm/pkg/proto/hapi/release"
 )
 
+type domains struct {
+	Enabled     bool                     `json:"enabled"`
+	DomainsList []map[string]interface{} `json:"domainsList"`
```

Comment:
typo, should be `domainList`

Reply:
Nope, It's ok.
https://github.com/helm/charts/tree/master/stable/traefik#lets-encrypt-wildcard-certificate